packages/api-explorer/__tests__/JsonForm/resolveReference.test.jsx

```diff
@@ -0,0 +1,501 @@
+/* eslint-disable no-use-before-define */
+import resolveReference from '../../src/JsonForm/resolveReference'
+
+test('resolve correctly', async () => {
+    const resp = await resolveReference(SCHEMA_BUG)
+    expect(resp).toEqual(EXPECTED_SCHEMA_BUG)
+})
+
+
+
+const SCHEMA_BUG = {
```

Comment:
These schema are the same of other JsonForm/index.test.js file?

Reply:
We should place this in an external file

Reply:
Are different case of schema